src/store/mod.rs

```diff
@@ -91,6 +93,21 @@ pub enum StoreConfigDataVersion {
 
 const DEFAULT_STORE_CONFIG_DATA_VERSION: u32 = StoreConfigDataVersion::Two as u32;
 
+#[derive(Clone, Debug, Serialize, Deserialize, Default)]
+pub struct ReplicaConfig {
```

Comment:
you could implement `From<PathBuf> for ReplicaConfig` setting a single offset at `0` to keep easily supporting the existing usage.